### DISCUSSION TITLE
3D registration monai

### DISCUSSION THREAD

**Discussion by kiwibruyant** [Project-MONAI/MONAI#4151] [Q&A] [answered]
Hi,

In the following tutorial [https://github.com/Project-MONAI/tutorials/blob/master/3d_registration/paired_lung_ct.ipynb](url) 3D CT scan volumes are registered but they have the same size 96x96x104. I'm working with a dataset of 3D ct scans of 512x512xZ with Z that is different for each patient. Does monai 3d registration module work in this case ? Or should I downsample my volumes in order to have the same z-coordinate for all of them ?

**Comment by Nic-Ma** [comment]
Hi @kate-sann5100 ,

Could you please help share some comments about this question?

Thanks in advance.

**Comment by wyli** [comment, marked as answer]
I think you can use pad/crop transforms to make the input shapes consistent. https://docs.monai.io/en/stable/transforms.html#crop-and-pad-dict
